scripts/scil_visualize_connectivity.py

```diff
@@ -184,6 +189,22 @@ def main():
     else:
         plt.savefig(args.out_png, dpi=300, bbox_inches='tight')
 
+    if args.histogram:
+        fig, ax = plt.subplots()
+        if args.exclude_zeros:
+            min_value = EPSILON
+        N, bins, patches = ax.hist(matrix.ravel(),
+                                   range=(min_value, matrix.max()))
+        nbr_bins = len(patches)
+        color = plt.cm.get_cmap(args.colormap)(np.linspace(0, 1, nbr_bins))
+        for i in range(0, nbr_bins):
+            patches[i].set_facecolor(color[i])
+
+        if args.show_only:
+            plt.show()
+        else:
+            plt.savefig(args.histogram, dpi=300, bbox_inches='tight')
+
```

Comment:
Any chance we can add a binning option? 
For example, this call
 scil_visualize_connectivity.py CM/track_count.npy tc.png --show_only --log --histogram tc_hist.png --exclude_zeros 
looked really good.

But this call looked pretty bad. I would have like to reduce the bin size to see better around 0 and 1000.
scil_visualize_connectivity.py CM/track_count.npy tc.png --show_only  --histogram tc_hist.png --exclude_zeros

![tc_hist](https://user-images.githubusercontent.com/2170887/78504471-84064280-773b-11ea-9b3f-f4e6ce9ee158.png)